currencies/rate_converter.go

```diff
@@ -172,11 +172,15 @@ func (rc *RateConverter) Rates() Conversions {
 
 // GetInfo returns setup information about the converter
 func (rc *RateConverter) GetInfo() ConverterInfo {
+	var rates *map[string]map[string]float64
+	if rc.Rates() != nil {
+		rates = rc.Rates().GetRates()
+	}
 	return converterInfo{
 		source:           rc.syncSourceURL,
 		fetchingInterval: rc.fetchingInterval,
 		lastUpdated:      rc.LastUpdated(),
-		rates:            rc.Rates().GetRates(),
+		rates:            rates,
 	}
```

Comment:
Since `rc.Rates()` can return `nil`, the `rates` field in the `ConverterInfo` object we return from the `GetInfo()` function can be `nil` too. Do we want that? Isn't that a risk of another null pointer dereference? After all, the `(ci converterInfo) Rates()` function just returns `rates` without checking for nil either. Are we ok with letting the caller check for `nil` on `Rates()` return value?
```
// Rates returns converter's internal rates
func (ci converterInfo) Rates() *map[string]map[string]float64 {
        return ci.rates
}
endpoints/currency_rates.go
```

Reply:
I think it's ok. This one is used in the info handler only. Having nil rates is valid but we should make sure it doesn't create a null ref. 
Before this patch, it was creating a null ref which cannot be handled by the `GetInfos` caller.

Reply:
Makes sense. Thank you @benjaminch 
Approved